backend/services/UserService.js

```diff
@@ -1,48 +1,18 @@
 const BaseModelService = require('./BaseModelService');
 
 class UserService extends BaseModelService {
-    constructor(bcryptjs) {
-        super();
-        this.bcryptjs = bcryptjs;
-    }
-
     get table() {
         return 'users';
     }
 
-    register(email, password) {
-        const salt = this.bcryptjs.genSaltSync(10);
-        const encryptedPassword = this.bcryptjs.hashSync(password, salt);
-
-        return this.create({
-            email,
-            password: encryptedPassword,
-            createdAt: Date.now()
-        });
-    }
-
-    async login(email, password) {
-        const user = await this.findOne({ email });
-
-        if (!user) {
-            return null;
-        }
-
-        if (this.bcryptjs.compareSync(password, user.password)) {
-            return this.getById(user.id);
-        }
-
-        return null;
-    }
-
     async getById(id, excludePassword = true) {
```

Comment:
just call it `findById`

Reply:
I would rather keep "findXYZ" ie. "findById" for global methods in abstract repository 
getXYZ would be our custom methods

Reply:
but then you have to remember that some of repos has `get` instead of `find` and I think e.g. in this case we do not need findById if we had to add another method for fetching by ID

Reply:
but if you are extend some methods you need to keep some rules
and we have also IDE for helping us which methods we have in which repositories

Reply:
basically we can overwrite findById

Reply:
done